### PR TITLE
Add Docs for enabled option for Java SDK

### PR DESCRIPTION
**Pull request by lbloder**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [x] Checked Vercel preview for correctness, including links
- [x] PR was reviewed and approved by any necessary SMEs
- [x] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

Documents changes done in https://github.com/getsentry/sentry-java/pull/2840

## Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## Extra resources

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

File: src/platforms/java/common/configuration/index.mdx [hunk 1]
```diff
@@ -297,6 +297,22 @@ SENTRY_DEBUG=true
 sentry.debug=true
 ```
 
+### Disable Sentry
+
+To disable Sentry in the debug mode, use the `enabled` option:
```
Comment (vivianyentran) [inline src/platforms/java/common/configuration/index.mdx:302]:
```suggestion
To disable Sentry in debug mode, use the `enabled` option:
```

**Review by vivianyentran — APPROVED** [review]
LGTM, thanks for updating!

File: src/platforms/java/common/configuration/index.mdx (hunk 1 above)
Comment (lbloder) [reply src/platforms/java/common/configuration/index.mdx:302]: Thank you for reviewing. Found a technical issue with that sentence and fixed it.

**Review by adinauer — APPROVED** [review]
LGTM

**Pull request merged**